src/encoded/search_views.py

```diff
@@ -42,6 +42,7 @@ def includeme(config):
     config.add_route('matrix', '/matrix{slash:/?}')
     config.add_route('reference-epigenome-matrix', '/reference-epigenome-matrix{slash:/?}')
     config.add_route('entex-matrix', '/entex-matrix{slash:/?}')
+    config.add_route('target_matrix', '/target-matrix{slash:/?}')
```

Comment:
Might want to change this to `target-matrix` to match `entex-matrix` etc. Not sure it really matters as long as we aren't using underscores in the URL.